=====================================
Global Biofield (GBF) Grand Challenge
=====================================

A vast river of viruses, phages, and virus-like particles—the **biofield**—flows around through each of us.  The biofield is transmitted through ingestion of foods, liquids, and aerosols.  Parts of the biofield are selectively amplified by interactions with hosts.  Readily carried intercontinentally by winds aloft, an estimated 9 billion biofield agents fall on each square meter of earth each day.  Airborne flux may play a part in global transmission of infectious diseases.  Biofield agents are also the largest single reservoir of carbon in the world’s oceans, with some 150 GT of them circulating.  Only a minuscule fraction of the biofield are viruses infectious to humans, while a slightly larger fraction infects non-human eukaryotic hosts.  But the bulk of the biofield consists of agents whereby bacteria carry out horizontal gene transfer and conduct warfare on each other.  These agents can have significant impacts on human health through influencing the ecology of the gut microbiome, which is known to play roles in antibiotic resistance, obesity, infection, and inflammation.  A recent survey found over 2000 strong associations between phages and human diseases.  While we know a lot about viruses that infect humans, there are a lot of open questions about the biofield as a whole.  On balance, is the biofield a positive force for health?  What are the barriers to biofield agents jumping hosts? How is the biofield is changed by urbanization and climate change?  Is there an biofield associated with wellness?  The biofield can be quantified and studied through metagenomic sequencing of stool samples from individuals, families, social groups, as well as environment samples.   **Our Grand Challenge is to characterize the biofield associated with human health and disease around the globe.**

See the accompanying `wiki <https://github.com/GlobalBiofield/globalbiofield.github.io/wiki/Global-Biofield-Grand-Challenge-Overview>` for a series of questions and answers.

Aims
----
1. Enable ML-driven hypothesis discovery of the interplay among human disease, wellness, and environment as reflected in the hierarchically-sampled biofield of global (mostly rural) populations.
2. Create an indexed, searchable corpus of biofield-associated sequence, health, environmental, video, and textual data that can be mined for knowledge about the agriculture, food, and culture of the people studied.
3. Create a cadre of trained “Global Biofield Scholars”, knowledgeable of local conditions, that are able to collect data, interact with clinicians and the public in their countries, and interpret results in terms of phylogenomic diversity, divergence, and pathogenicity.

Technical Approach
------------------
Metagenomic biofield samples from clinics, families, and individuals published along with de-identified health and environmental metadata, video of local food markets and drone footage, searchable by indices of sequence, video, and essays.  There is a major decision to be made—elided for now—about on-site nanopore sequencing versus short-read sequencing at regional facilities, with arguments to be made for both on the bases of cost, technical merit, and human factors.  Another major decision is whether to included RNA viruses as well as DNA viruses.

Scale
-----
Assuming sequencing costs can be driven from $200/sample in the first year to around $100/sample in the subsequent years, then funding at a level of $4.8M/year for 4 years would result in roughly 50 Scholars from 25 countries collecting resulting in 8 to 30 TB of indexed biofield data from 17,000 global samples with associated health metadata, video, and essays.

Ethics Core
-----------
We envision that Ethics drives the data acquisition process.  This Core would be best led by a scientist with strong credentials in global health, health records, and local conditions.   The Ethics Core will select which countries (other then the US) which be G countries, and deliver deep knowledge on potential host clinics as well as health and infectious-disease issues for each country chosen.   They will direct each Scholar’s journey in their country to maximize likely biofield diversity and conduct retrospective analyses to improve strategy.  Ethics will use local laws and customs as well as informed-consent knowledge, to establish procedures for consent and de-identification of data.  The Ethics Core will also determine if and when local conditions warrant redirecting a Scholar from the survey activities to a developing local health crisis if their skills would prove useful there.

Workforce Development
---------------------
We envision Workforce Development Core being led by a physician/scientist skilled in training and curriculum development for global health professionals. Each year, they will select 7 to 15 Global Biofield Scholars from 2 to 12 GBF countries, with written communication ability in English a strong selection criterion. Each Scholar will be trained to carry out all operations: sampling, sample prep, sequencing, metadata collection, data deposition, writing essays on collection, and simple interpretation of results.  Development will emphasize the production of self-reliant Scholars whose curiosity drives their work. Scholars from previous years may be added to the training team for future years, using their field knowledge to improve training.  

Teaming
-------
We envision Teaming Core to be led by a scientist with strong knowledge of in-country clinic practice and conditions as well as teleclinical work.  Teaming will select local clinics that can host Scholars, and will supervise interactions among Scholars, clinicians, schools, and the public.  each week to collect roughly average, 1 sample per day, in addition to getting consent, training clinicians, writing essays, taking video, and interacting with selected families, individuals, and schools.  Teaming will work to identify, recognize, and reward Scholars that perform exceptionally well, and to make the project fun by recognizing milestones achieved.  Teaming also has an important role in engagement with clinician and public-health user groups.

Data Collection
---------------
We envision the Data Collection Core to be led by a scientist with a strong record in viral metagenomics.  Early on, this core will be focused on the technical means of sampling and sequencing to find the best compromise among costs, simplicity, data depth/length, and Scholar workload.  This core will have strong pressures from sequencing costs, and potential cost sharing as well as public export of equipment and protocols with a sequencing vendor might be important.

Standards
---------
We envision Standards Core to be led by a physician/scientist with a strong record in leading ML-related Common Fund activities.   Standards has the critical role of maintaining dictionaries, depositing data and metadata into appropriate repositories, and enabling access to data.  Standards has responsibility for the interface to the ML community, maintains the resources for cloud access of GBF data, and runs the annual data jamboree. Standards acts as the voice for the ML consumer, balancing the simplicity that uniform data collection strategies offer against hybrid data-collection approaches that get results more quickly but result in headaches for future ML data consumers.  

Tool Development
----------------
We envision the Tool Development Core to be led by a scientist with a strong record in fielding software for phylogenetic analysis that provide immediate feedback on sample quality, diversity, divergence, and phylogenetic distribution.  Because Bridge2AI is aimed at hypothesis generation and not hypothesis testing, maximizing the novelty/surprise in samples is a priority, so tools to assess this will need to run quickly on laptops in the field.  This Core will develop ML tools that Ethics will use to direct sampling activities. Tool Development will also conduct research and write software that enables efficient search of biofield sequences via indices of sequence-based signatures.
